internal/pkg/config/v1/config.go

```diff
@@ -127,6 +128,7 @@ func New() *Config {
 		Contexts:      make(map[string]*Context),
 		ContextStates: make(map[string]*ContextState),
 		AnonymousId:   uuid.New().String(),
+		Salt:          uuid.New().String(),
```

Comment:
Is this a best practice for generating the salt? Is it pseudorandom generator or does it generate IDs predictably?

Reply:
@ankurjoshi9 wdyt?

Reply:
Go library google/uuid is based on [RFC 4122](https://www.rfc-editor.org/rfc/rfc4122#section-6). It is not recommended to use for security capabilities. 
Use https://pkg.go.dev/crypto/rand for Salt generation.